lib/src/main/kotlin/com/swmansion/starknet/provider/gateway/GatewayProvider.kt

```diff
@@ -440,7 +440,7 @@ class GatewayProvider(
 
     companion object Factory {
         @JvmStatic
-        fun makeTestnetClient(): GatewayProvider {
+        fun makeTestnetProvider(): GatewayProvider {
```

Comment:
This is a breaking change. Please point the target branch to the 4.0.0 changes (it might need an update from main).